src/app/shared/interceptors/http.token.interceptor.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable, Injector } from '@angular/core';
+import { HttpEvent, HttpInterceptor, HttpHandler, HttpRequest } from '@angular/common/http';
+import { Observable } from 'rxjs/Observable';
+
+import { JwtService } from '../services';
+
+@Injectable()
+export class HttpTokenInterceptor implements HttpInterceptor {
+  constructor(private jwtService: JwtService) {}
+
+  intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
+    const headersConfig = {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+    };
+
+    if (this.jwtService.getToken()) {
```

Comment:
no need to call `getToken` twice here, put the result in a local variable

Reply:
done